back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -155,7 +155,7 @@ def create
     extract_custom_field_values_from_params!(participation_method.custom_form)
     params_for_create = idea_params participation_method.custom_form, is_moderator
     input = Idea.new params_for_create
-    input.creation_phase = (phase if participation_method.creation_phase?)
+    input.creation_phase = (phase if !participation_method.transitive?)
```

Comment:
Transitive is proxy for ideation, correct? The other methods are not transitive?

Reply:
Yes, but besides ideation, voting is also transitive.

Reply:
I know this is out of scope for this PR, but note to self while thinking about all of this: Wouldn't it be easier to always save the creation phase?

Reply:
Yes, it would feel cleaner to me if we would always save the `creation_phase`. The difficulty now is migrations (we would probably set it to the first of the associated phases?).

Today, we will very likely introduce a `transitive` scope, corresponding to all inputs with no creation_phase (used for the global input manager that will only list ideation inputs). But it would be good if we could think about some good abstractions to filter the different inputs, while avoiding explicit "Idea", "Proposal" etc. types.